assets/post_install_macOS.sh

```diff
@@ -45,6 +45,9 @@ ${DSTROOT}/.mne-python/bin/conda env config vars set PYTHONNOUSERSITE=1
 logger -p 'install.info' "ℹ️ Disabling mamba package manager banner"
 ${DSTROOT}/.mne-python/bin/conda env config vars set MAMBA_NO_BANNER=1
 
+logger -p 'install.info' "ℹ️ Configuring Matplotlib to use the Qt backend by default"
+${DSTROOT}/.mne-python/bin/conda env config vars set MPLBACKEND=qtagg
```

Comment:
I'd rather not use an env var because it actually takes precedence over `matplotlibrc`. If possible a better option would be to use the lowest-priority location, which I guess would be (4) here

https://matplotlib.org/stable/tutorials/introductory/customizing.html#the-matplotlibrc-file

Reply:
I'd actually **intended** to override the `matplotlibrc`, hence picked the env var 🤔 

Another approach I'd considered is deploying our own MPL config and setting [`MPLCONFIGDIR`](https://matplotlib.org/stable/users/faq/environment_variables_faq.html#envvar-MPLCONFIGDIR), so users would still be able to alter the default via the Matplotlib Python API

WDYT?

Reply:
I would rather not do that. I think a user's matplotlibrc should take precedence if they have one. It allows us to respect the standard order/expectations as much as possible.

Hence 4 in the link above is best I think

Reply:
... let's say for example for 10 years I've been using `Agg` as my backend by setting it in `~/.matplotlilb/matplotlibrc` (for whatever reason). This would have worked on every platform and in every install automatically. What you currently do here will make it (mysteriously!) stop working. Same thing if we go the MPLCONFIGDIR route (and maybe worse, this approach would ignore/break any *other* config changes people have in there).

Using option (4) above respects it. Hence it's the least invasive (and therefore best) option I think to use (4).

Basically in my mind we *just* want to change the internal matplotlib default from MacOS to Qt. We can't/shouldn't patch MPL's code, so we should instead make the next least invasive / pushy change, which is to ship a `matplotlibrc` in the `matplotlib` installation directory.

Reply:
Ok, convinced